.travis.yml

```diff
@@ -31,14 +31,15 @@ matrix:
   include:
     #clang check
     - name: "clang-format Check"
-      os: osx
+      os: linux
       compiler: clang
       before_script:
-        - brew install clang-format
+        - sudo apt-get -q update
+        - sudo apt-get -y install clang-format
         - mkdir build && cd build && cmake .. -DCMAKE_BUILD_TYPE=Debug -DBUILD_TEST=TRUE
       script:
         - cd ..
-        - bash scripts/check-clang.sh
+        - ./scripts/check-clang.sh
```

Comment:
Any reason for this change, there is a difference here in terms of permissions that need to be set on the file. (Some details here: https://unix.stackexchange.com/questions/136547/what-is-the-difference-between-running-bash-script-sh-and-script-sh). In any case not a blocking change just wanted to understand.

Reply:
No. Will change it back to bash once CI passes. I misunderstood the reason for this travis job failure and changed to ./. Also, travis check is stuck at "waiting for status to be reported..." 😔

Reply:
Moved to bash. Merging.